test/functional/cmdLineTests/shrcdbgddrext/shrcdbgextddrtests.xml

```diff
@@ -139,7 +139,7 @@
                 <input>!shrc stats</input>
                 <input>quit</input>
         </command>
-  <output regex="no" type="success" >4 orphans</output>
+  <output regex="yes" javaUtilPattern="yes" type="success" >([4-9]|[1-9][0-9]+) orphans</output>
```

Comment:
Can you add a comment above this line saying that there are additional `java/lang/invoke/BoundMethodHandle$Species` classes in OJDK MH builds so that > 4 case is needed ?

Reply:
Added.